content/en/about/feature-stages/index.md

```diff
@@ -59,6 +59,12 @@ Below is our list of existing features and their current phases. This informatio
 | [Trace Sampling](/docs/tasks/observability/distributed-tracing/configurability/#trace-sampling) | Beta
 | [Request Classification](/docs/tasks/observability/metrics/classify-metrics/) | Beta
 
+### Extensibility
+
+| Feature           | Phase
+|-------------------|-------------------
+| [WebAssembly Extension](/docs/ops/configuration/extensibility/wasm-module-distribution) | Alpha
```

Comment:
I does seem strange to have an Alpha feature have a link that leads to an Experimental page.

Reply:
I removed the doc link. We don't really have any other doc about Wasm except this linked one, or a blog post created back in 1.5 where we claimed alpha support for Wasm. Looks like there are other features in this page which does not have link attached, so I simply remove the link.